app/controllers/application_controller/tree_support.rb

```diff
@@ -20,6 +20,11 @@ def squash_toggle
     end
   end
 
+  def find_record
+    identify_record(params[:id], controller_name.camelize) if %w(container_image host).include? controller_name
```

Comment:
Won't `controller_name.camelize` resolve to a string, whereas previously, we passed the Host class?